systemd/ua-auto-attach.service

```
@@ -1,10 +1,10 @@
 [Unit]
 Description=Ubuntu Advantage auto attach
-Before=cloud-config.service
+After=cloud-config.target
```

We need to keep the before line to ensure cloud-init waits for this service to finish before it runs its `ubuntu_advantage` module or `package_upgrade` module etc
```suggestion
After=cloud-config.target
Before=cloud-config.service
```